collections/Bodl/MS_Bodl_738.xml

```diff
@@ -62,29 +129,29 @@
+                  <provenance><orgName key="org_127815410" role="fmo">Exeter, Devon, Cathedral church of St Peter</orgName>. Registered in the library inventory of 1506 of Exeter cathedral among the books of <q>Decimus descus</q>, <q>in parte orientali</q>: <q>Glosa super Psalterium in exposicione literali Nicholai <sic> Treneth</sic>, 2 fo., Quia ternarius</q>. (Oliver, <title>Lives of the Bishops of Exeter</title>, p. 373; Willoughby and Ramsay, <title>Libraries of the Secular Cathedrals</title>, SC35.519.)</provenance>
+                  <acquisition when="1602">The manuscript was presented to <persName key="person_34512366">Sir Thomas Bodley</persName> by the Dean and Chapter of Exeter in 1602. (Oliver, <title>Lives of the Bishops of Exeter</title>, p. 377; cf. Erskine, <q>The growth of Exeter Cathedral library</q>, p. 49.)</acquisition>
                </history>
                <additional>
                   <adminInfo>
                      <recordHist>
-                        <source>
-                           <ref target="https://medieval.bodleian.ox.ac.uk/about">Summary description</ref> abbreviated from the Summary Catalogue (1922). Decoration, localization and date follow Pächt and Alexander (1973). <listBibl>
-                              <bibl facs="aap0539.gif" type="SC">Summary Catalogue, vol. 2, part 1, p. 518</bibl>
-                              <bibl facs="aap0540.gif" type="SC">Summary Catalogue, vol. 2, part 1, p. 519</bibl>
-                           </listBibl>
-                        </source>
+                        <source>Description by Jakub Kujawiński, <ref target="https://doi.org/10.1484/M.IPM-EB.5.138134">
+                              <title>Nicholas Trevet’s Commentary on the Psalms (1317–c. 1321): A Publishing History</title>
+                           </ref> (Turnhout: Brepols, 2023), 187–195. Adapted by Andrew Dunning (2024).</source>
                      </recordHist>
```

Comment:
our usual policy is to keep to SC references here (preceded by "Previously described in the Summary Catalogue" or similar - any good reason to change this?